sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_version.py

```diff
@@ -1,6 +1,9 @@
-# ------------------------------------
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-# ------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b1"
```

Comment:
this version should go up to `1.0.0b3`

Reply:
latest `azure-messaging-webpubsubservice` is `1.0.0b1`, so next version should be `1.0.0b2`.  https://pypi.org/project/azure-messaging-webpubsubservice/#history